TimeTrackerUI/MainWindow.xaml.cs

```diff
@@ -24,30 +28,140 @@ public partial class MainWindow : Window
         public MainWindow()
         {
             InitializeComponent();
-            //CurrentTimeLabel.Content = DateTime.Now.ToString("MM/dd/yyyy HH:mm");
         }
 
         private void Window_Loaded(object sender, RoutedEventArgs e)
+        {
+            ShowDateTime();
+            LoadComboboxItems();
+
+            _tasks = GetTasks();
```

Comment:
No. Think it better.How can you do it? You have the _tasks here because you want to access then and modify , but why ? is there not any other way ?